content/microservices/basics.md

```diff
@@ -348,3 +348,27 @@ export interface RequestContext<T = any> {
 ```
 
 The `data` property is the message payload sent by the message producer. The `pattern` property is the pattern used to identify an appropriate handler to handle the incoming message.
+
+
+### Handling timeouts
+
+Sometimes, microservices are down or not available. You can handle these cases using the `RxJS` timeout operator in the call. If the microservice does not respond to the request within a certain time, an Exception is thrown, which can be caught and handled appropriately.
+
+To solve this problem you have to use [rxjs](https://github.com/ReactiveX/rxjs) package. Just use the `timeout` operator in the pipe:
```

Comment:
Can we get consistent use of `RxJS` on the page? Also, can we say `you have to use the RxJS package`?